requirements/tests.txt

```diff
@@ -0,0 +1,13 @@
+asynctest
+codecov
+flake8
+interrogate
+isort
+# Note: used for kwarray.group_items, this may be ported to mmcv in the future.
+kwarray
+pytest
+pytest-cov
+pytest-runner
+ubelt
```

Comment:
`kwarray`, `pytest-cov`, `pytest-runner` and `ubelt` are unnecessary.

Reply:
unnecessary packages have been removed